### PR TITLE
Update 1.what-is-cloud.md

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/nebula-cloud/1.what-is-cloud.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Nebula Graph Cloud (Cloud for short) is a product that supports the [Azure](https://azure.microsoft.com/zh-cn/) platform to build a Nebula Graph database on the cloud and supports one-click deployment of Nebula Graph. You can create a graph database within minutes, and quickly expand computing and storage.
+
+## Scenarios
+
+You want to use Azure cloud resources to quickly subscribe to the Nebula Graph service and to deploy the Nebula Graph database to the cloud, supporting the use of visualized tools and functions of privilege management.
+
+## Product features
+
+- Nebula Graph Cloud is a pay-as-you-go and pay-on-demand subscription. You can create or stop Nebula Graph instances at any time according to your own business development, which shows the full flexibility of cloud services.
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/1.what-is-cloud.md:11]: Is it fair to say that Nebula Graph Cloud is a subscription?

File: docs-2.0/nebula-cloud/1.what-is-cloud.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Integrated visualized database management tools include Nebula Graph Studio, Nebula Dashboard, and Nebula Explorer. You can use them to import graph datasets, execute nGQL statements explore graph data, monitor data, etc. Even without any experience in graph database operation, you can quickly become a graph expert. For details, see [Supporting applications](../nebula-cloud/5.solution/5.1.supporting-application.md).
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/1.what-is-cloud.md:12]:
```suggestion
- Integrated visualized database management tools include Nebula Graph Studio, Nebula Dashboard, and Nebula Explorer. You can use them to import graph datasets, execute nGQL statements to explore graph data, monitor data, etc. Even without any experience in graph database operation, you can quickly become a graph expert. For details, see [Supporting applications](../nebula-cloud/5.solution/5.1.supporting-application.md).
```

File: docs-2.0/nebula-cloud/1.what-is-cloud.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- You can create private links to connect to the cloud database using Console without having access to the cloud platform. For details, see [Private Link](../nebula-cloud/5.solution/5.2.connection-configuration-and-use.md).
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/1.what-is-cloud.md:13]:
```suggestion
- You can create private links to connect to the cloud database using Nebula Console without having access to the cloud platform. For details, see [Private Link](../nebula-cloud/5.solution/5.2.connection-configuration-and-use.md).
```

File: docs-2.0/nebula-cloud/1.what-is-cloud.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Product features
+
+- Nebula Graph Cloud provides a pay-as-you-go and pay-on-demand subscription. You can create or stop Nebula Graph instances at any time according to your own business development, which shows the full flexibility of cloud services.
+- Integrated visualized database management tools include Nebula Graph Studio, Nebula Dashboard, and Nebula Explorer. You can use them to import graph datasets, execute nGQL statements explore graph data, monitor data, etc. Even without any experience in graph database operation, you can quickly become a graph expert. For details, see [Supporting applications](../nebula-cloud/5.solution/5.1.supporting-application.md).
+- You can create private links to connect to the cloud database using Console without having access to the cloud platform. For details, see [Private Link](../nebula-cloud/5.solution/5.2.connection-configuration-and-use.md).
+- Access control. You can add database users with different role privileges to the specified graph space to ensure business data security.
+- Official support. You can quickly create orders for technical support, consulting problems you have encountered during use.
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/1.what-is-cloud.md:15]:
```suggestion
- Official support. You can quickly create tickets for technical support, consulting problems you have encountered during use.
```

File: docs-2.0/nebula-cloud/1.what-is-cloud.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+## Product advantages
+
+- Flexible and efficient: You can quickly build a Nebula Graph database without purchasing servers, storage devices, computing, and network components. It only takes 5-10 minutes to prepare a complete environment for use.
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/1.what-is-cloud.md:19]: Actually, users have to purchase virtual servers on Nebula Graph cloud.

File: docs-2.0/nebula-cloud/1.what-is-cloud.md (hunk 4 above)
Comment (izhuxiaoqing) [reply docs-2.0/nebula-cloud/1.what-is-cloud.md:15]: 和小狼确认了一下，`工单`的翻译最后用了order

**Review by randomJoe211 — APPROVED** [review]

File: docs-2.0/nebula-cloud/1.what-is-cloud.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+Nebula Graph Cloud (Cloud for short) is a product that supports one-click deployment of Nebula Graph on the [Azure](https://azure.microsoft.com/en-us/) platform. You can create a graph database within minutes, and quickly expand computing and storage.
+
+## Scenarios
+
+You want to use Azure cloud resources to quickly subscribe to the Nebula Graph service and to deploy the Nebula Graph database to the cloud, supporting the use of visualized tools and functions of privilege management.
+
+## Product features
+
+- Nebula Graph Cloud provides a pay-as-you-go and pay-on-demand subscription. You can create or stop Nebula Graph instances at any time according to your own business development, which shows the full flexibility of cloud services.
+- Integrated visualized database management tools include Nebula Graph Studio, Nebula Dashboard, and Nebula Explorer. You can use them to import graph datasets, execute nGQL statements, explore graph data, monitor data, etc. Even without any experience in graph database operation, you can quickly become a graph expert. For details, see [Supporting applications](../nebula-cloud/5.solution/5.1.supporting-application.md).
```
Comment (abby-cyber) [inline docs-2.0/nebula-cloud/1.what-is-cloud.md:12]:
```suggestion
- Integrated visualized database management tools include Nebula Graph Studio, Nebula Dashboard, and Nebula Explorer. You can use them to import graph datasets, execute nGQL statements, explore graph data, monitor data, etc. Even without any experience in graph database operations, you can quickly become a graph expert. For details, see [Supporting applications](../nebula-cloud/5.solution/5.1.supporting-application.md).
```

File: docs-2.0/nebula-cloud/1.what-is-cloud.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+- You can create private links to connect to the cloud database using Nebula Console without having access to the cloud platform. For details, see [Private Link](../nebula-cloud/5.solution/5.2.connection-configuration-and-use.md).
```
Comment (abby-cyber) [inline docs-2.0/nebula-cloud/1.what-is-cloud.md:13]:
```suggestion
- You can create private endpoints to connect to the cloud database using Nebula Console without having access to the cloud platform. For details, see [Private Link](../nebula-cloud/5.solution/5.2.connection-configuration-and-use.md).
```

**Review by abby-cyber — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**